### PR TITLE
add feature from features/rickroll

### PR DESCRIPTION
**Pull request by narawat-surattikan**

### CODE REVIEW

**Review by houtarouhyouka1203 — CHANGES_REQUESTED** [review]
add smth in title tag

File: rickroll/index.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title></title>
+    <title>r i c k r o l l</title>
```
Comment (houtarouhyouka1203) [inline rickroll/index.html:6]: fix the title content to ```rickroll```, not ```r i c k r o l l```

**Review by houtarouhyouka1203 — CHANGES_REQUESTED** [review]
fix the title. see the line 6

File: rickroll/index.html (hunk 1 above)
Comment (narawat-surattikan) [reply rickroll/index.html:6]: ill fix and make sure to push another commit

**Review by narawat-surattikan — COMMENTED** [review]
ill fix it dw

**Review by houtarouhyouka1203 — APPROVED** [review]
OK. that looks fine

**Pull request merged**